iscript/src/iscript/mac.py

```diff
@@ -194,7 +194,7 @@ async def sign_single_files(config, sign_config, all_paths):
     """
     identity = sign_config["identity"]
     keychain = sign_config["signing_keychain"]
-    sign_command = _get_sign_command(identity, keychain, sign_config)
+    sign_command = _get_sign_command(identity, keychain, sign_config, file_=all_paths[0].single_file_globs[0])
```

Comment:
Hm, are we sure we'll never have multiple `all_paths`?
If so, we may want to move this down into the 2nd `for app in all_paths:` below.

Reply:
I am pretty sure -- this is the single_file case after all. But I still like the idea - thanks!